app/gui/src/project-view/util/codemirror/index.ts

```diff
@@ -106,14 +101,8 @@ export function useCodeMirror(
   }
 }
 
-function useBindings({
-  readonly,
-  contentDOM,
-}: {
-  readonly: Readonly<Ref<boolean>>
-  contentDOM: HTMLElement
-}) {
-  const keyboard = injectKeyboard()
+function useBindings(view: EditorView) {
+  const keyboard = injectKeyboard(true)
```

Comment:
Perhaps we could also move `provideKeyboard` from GraphEditor.vue to App.vue, and should be then visible in React.